docker-compose-cthulhu.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.3"
+
+services:
+  onboard-cthulhu:
+    image: dukerobotics/robosub-ros:onboard
+    container_name: onboard-cthulhu
+    environment:
+      - ROBOT_NAME=cthulhu
```

Comment:
Is the only difference in these compose files the existence of these environment variables? If so is there some other way to inject the robot name into the container without needing duplicate compose files? Maybe we can refactor our code to use [Extension](https://docs.docker.com/compose/extends/#extending-services).

Reply:
Made that change